drivers/serial/Kconfig.serial_shell

```
@@ -0,0 +1,37 @@
+# Copyright (C) 2024 Bang & Olufsen A/S, Denmark
+# SPDX-License-Identifier: Apache-2.0
+
+config SERIAL_SHELL
+	bool "serial shell"
+	depends on SHELL
+	depends on UART_INTERRUPT_DRIVEN
+	depends on LOG
+	default n
+	help
```

It should not depend on `LOG` and `default n` can be omitted.

```suggestion
config SERIAL_SHELL
	bool "Serial shell"
	depends on SHELL
	depends on UART_INTERRUPT_DRIVEN
	help
```